mozilla_django_oidc/auth.py

```diff
@@ -79,10 +79,8 @@ def create_user(self, claims):
     def verify_token(self, token, **kwargs):
         """Validate the token signature."""
         nonce = kwargs.get('nonce')
-
         secret = self.OIDC_RP_CLIENT_SECRET
-        if import_from_settings('OIDC_RP_CLIENT_SECRET_ENCODED', False):
-            secret = base64.urlsafe_b64decode(self.OIDC_RP_CLIENT_SECRET)
+
         # Verify the token
         verified_token = jws.verify(token, secret, algorithms=['HS256'])
```

Comment:
nit: You can directly use here `self.OIDC_RP_CLIENT_SECRET` since `secret` it's not used anywhere else now that you are removing the base64 stuff so that we can remove another line.